notebooks/mounts_listing.py

```diff
@@ -0,0 +1,83 @@
+# Databricks notebook source
```

Comment:
why do we need this file? we should have a task in the `assessment` step and that's it.

Reply:
I've created this file for testing purposes, it allowed me to run test_installation without running other commands